.circleci/config.yml

```diff
@@ -5,7 +5,8 @@ jobs:
     docker:
       - image: circleci/ruby:latest
     steps:
-      - setup_remote_docker
+      - setup_remote_docker:
+          docker_layer_caching: true
```

Comment:
ah cool didn't know our plan covered this!

Reply:
Yeah nice little feature, keen to see the effects